test/base/EtherCollateralLoansTakeOutLoanTest.js

```diff
@@ -58,19 +61,26 @@ contract('EtherCollateralLoansTakeOutLoanTest', function (accounts) {
         from: owner,
         Mock,
         initialize: true,
-        onInitialize: async (instance, { escrowFactory, chainlinkAggregator }) => {
+        onInitialize: async (instance, { escrowFactory, chainlinkAggregator, atmSettings }) => {
           await escrowFactory.givenMethodReturnAddress(
             escrowFactoryInterfaceEncoder.encodeCreateEscrow(),
             createdEscrowMock.address
           )
           chainlinkAggregatorInstance = chainlinkAggregator
+          atmSettingsInstance = atmSettings
         }
       }, {
         [settingsNames.OverCollateralizedBuffer]: overCollateralizedBuffer,
         [settingsNames.CollateralBuffer]: collateralBuffer,
         [settingsNames.LiquidateEthPrice]: liquidateEthPrice,
       })
 
+    const atmGovernance = await Mock.new();
+    await atmSettingsInstance.givenMethodReturnAddress(
+      IAtmSettingsEncoder.encodeGetATMForMarket(),
+      atmGovernance.address
+    );
+
```

Comment:
why did we need to add this?

Reply:
To mock the atmMarket address in the test for the _isSupplyToDebtValid() check at the start of the takeOutLoan() method